src/tests/bulkConcepts/addBulkConcepts.test.jsx

```diff
@@ -51,4 +52,52 @@ describe('Add Bulk Concepts', () => {
     </MemoryRouter>);
     wrapper.find('#btn-add-all').at(0).simulate('click');
   });
+  it('it calls hanldleSelect', () => {
+    const props = {
+      fetchCielConcepts: jest.fn(),
+      addExistingBulkConcepts: jest.fn(),
+      cielConcepts: multipleConceptsMockStore.concepts.dictionaryConcepts,
+      isFetching: true,
+      match: { params: { type: 'users', typeName: 'mochu', collectionName: 'andela' } },
+    };
+
+    const wrapper = mount(<MemoryRouter>
+      <Provider store={store}>
+        <AddBulkConcepts {...props} />
+      </Provider>
+    </MemoryRouter>);
+
+    const event = {
+      preventDefault: jest.fn(),
+      target: {
+        value: ['/users/nesh/collections/test/concepts/'],
+        checked: true,
+      },
+    };
+
+    const spy = jest.spyOn(wrapper.find('AddBulkConcepts').instance(), 'handleSelect');
+    wrapper.instance().forceUpdate();
+    wrapper.find('#ciel').at(0).simulate('click');
+    wrapper.find('.table-check').at(0).simulate('change', { event });
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+
+  it('it calls hanldleBack', () => {
+    const props = {
+      fetchCielConcepts: jest.fn(),
+      addExistingBulkConcepts: jest.fn(),
+      cielConcepts: multipleConceptsMockStore.concepts.dictionaryConcepts,
+      isFetching: true,
+      match: { params: { type: 'users', typeName: 'mochu', collectionName: 'andela' } },
+    };
+    const wrapper = mount(<MemoryRouter>
```

Comment:
You have repeated some code in the above test. Please create a beforeEach and place the repeated code in there, implement DRY programming.

Reply:
Sure. Thank you for noticing that.